### DISCUSSION TITLE
How to ignore the case of file and directory names?

### DISCUSSION THREAD

**Discussion by ckafi** [carapace-sh/carapace-bin#1791] [Q&A] [answered]
Is there a way to get the completions for files and directories to ignore case?
E.g. the completion for `ls d` should produce both `Documents` and `dev`.

**Comment by rsteube** [comment, marked as answer]
> Not at the moment, no: https://github.com/rsteube/carapace/issues/757

Should now be possible by setting the environment variable `CARAPACE_MATCH` to `CASE_INSENSITIVE` or `1`

**Reply by ckafi** [reply]
Thanks for the answer.

**Reply by SanchithHegde** [reply]
Hey @rsteube, thanks for developing and maintaining this awesome tool. 

It would be great if this information about the environment variable for case-insensitive search could be included in the documentation. Do let me know if you'd like me to open a PR for including it, and which section of the documentation this can go in (I'm thinking the Setup or the Completers section?).

**Reply by rsteube** [reply]
Yes, `Setup` would be a good place for it. Pr is welcome.
